Source/UI/Scripts/d3/gamesPlayedPieChart.js

```diff
@@ -0,0 +1,17 @@
+﻿function drawPieChart(summariesOfGames) {
```

Comment:
this is a global function. We don't use them. It should be a namespace or a js module.